.github/workflows/run_tests.yml

```diff
@@ -108,6 +108,25 @@ jobs:
           echo "CXX=icpx" >> $GITHUB_ENV &&
           echo "FC=ifort" >> $GITHUB_ENV
 
+      # Set up perl environment for LCOV
+      - uses: actions/checkout@v3
+      - name: Setup perl
+        uses: shogo82148/actions-setup-perl@v1
+        with:
+          perl-version: '5.30'
+          install-modules: Memory::Process
```

Comment:
Any particular reason why this couldn't be installed with apt?

Reply:
library not found, so we had to do it this way